media-gfx/cura/cura-0.15.04.4.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit eutils fdo-mime gnome2-utils python-single-r1
+
+MY_PV=${PV#0.}
+MY_PN=Cura
+SRC_URI="https://github.com/daid/${MY_PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+DESCRIPTION="A mesh slicer written in python to produce gcode for 3D printers"
+HOMEPAGE="https://github.com/daid/Cura"
+LICENSE="AGPL-3"
+SLOT="0"
+IUSE=""
+
+RDEPEND="${PYTHON_DEPS}
+	dev-python/wxpython:3.0[opengl,${PYTHON_USEDEP}]
+	>=dev-python/numpy-1.6.2[${PYTHON_USEDEP}]
+	>=dev-python/pyopengl-3.0.2[${PYTHON_USEDEP}]
+	>=dev-python/pyserial-2.6[${PYTHON_USEDEP}]
+	>=media-gfx/curaengine-${PV}
+"
+DEPEND="${RDEPEND}
+	>=dev-python/setuptools-0.6.34[${PYTHON_USEDEP}]"
+
+REQUIRED_USE="${PYTHON_REQ_USE}"
+PATCHES=( "${FILESDIR}/${P}-nopower.patch" )
+S="${WORKDIR}/${MY_PN}-${MY_PV}"
+
+src_prepare() {
+	cat > "${T}"/cura <<- CURAEOF || die
+		#!/bin/sh
+		PYTHONPATH="\$PYTHONPATH:${EROOT}usr/share/cura/" "${PYTHON}" "${EROOT}usr/share/cura/cura.py" "\$@"
```

Comment:
Referencing ROOT in src_* phases is forbidden. See: https://devmanual.gentoo.org/ebuild-writing/variables/index.html#root

Unless you want to support prefix, you can safely assume that ROOT is always `/`.